vhdbuilder/packer/install-dependencies.sh

```diff
@@ -68,6 +68,8 @@ if grep -q "fullgpu" <<< "$FEATURE_FLAGS"; then
     ensureGPUDrivers
 fi
 
+installAscBaseline
```

Comment:
we should hold on the merge of this change until it's clear.

Reply:
At this current state it is experimental